monai/data/image_reader.py

```diff
@@ -844,21 +847,21 @@ def get_data(
 
         return patches, metadata
 
-    def _check_level(self, img, level):
-        level = self.level
+    def _get_max_level(self, img_obj):
+        """
+        Return the maximum number of levels in the whole slide image
+        Args:
+            img: the whole slide image object
 
-        level_count = 0
+        Returns:
```

Comment:
I feel no need this `returns` part as it's exactly same as the doc-string.

Thanks.